test/hotspot/jtreg/runtime/cds/appcds/customLoader/SameNameInTwoLoadersTest.java

```diff
@@ -54,7 +54,7 @@ public static void main(String[] args) throws Exception {
         customJar = JarBuilder.build("SameNameInTwoLoadersTest_custom", "CustomLoadee", "CustomLoadee3");
 
         useWbParam = "-Xbootclasspath/a:" +
-            JarBuilder.build(true, "WhiteBox", "sun/hotspot/WhiteBox");;
+            JarBuilder.build(true, "WhiteBox", "jdk/test/whitebox/WhiteBox");;
```

Comment:
New code has only one ';'. Feel free to remove one (accept new code). Not required.